mocks/mockoon.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
If a crumb of context is needed, see the new file `tests/integration/INTEGRATION_TESTING.md`